RDSC-3487: Update RDI -> Deploy a pipeline documentation (#1423)
redis/docs: content/integrate/redis-data-integration/data-pipelines/deploy.md

```

### Set secrets for K8s/Helm deployment
 
Use the provided `scripts/rdi-secret.sh` shell script to set the specified secrets. The general pattern to use it is:
```bash
scripts/rdi-secret.sh set <SECRET-KEY> <SECRET-VALUE>
```

> The script offers functionality to retrieve a specific secret, as well as the capability to list all available secrets within the system:

Reviewer: ```suggestion
The script lets you retrieve a specific secret or list all the secrets that have been set:
```
